cli/pkg/kctrl/cmd/package/builder/pkgBuilder.go

```diff
@@ -0,0 +1,17 @@
+package builder
+
+import (
+	"github.com/vmware-tanzu/carvel-kapp-controller/pkg/apiserver/apis/datapackaging/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type PkgBuilder struct {
+	metav1.TypeMeta `json:",inline"`
+	Config          Config `json:"config, omitempty"`
+}
+
+type Config struct {
+	Pkg         v1alpha1.Package         `json:"package"`
```

Comment:
should use pointers here probably so that they are seen as "optional" + omitempty as user fills out PackageBuild.

Reply:
Adopted